tests/unit/records/test_jobs.py

```diff
@@ -24,13 +26,12 @@ def mysource(a: int,
             self.assertEqual(google_cloud_creds, mock_session.creds.google_sheets.return_value)
             self.assertEqual(schema_name, 'myschema')
             self.assertEqual(spectrum_base_url, 'spectrumschema')
-            yield mock_source
+            return mock_source
 
-        @contextmanager
```

Comment:
Whoops!  This never got fixed when sources/targets were made to be eager rather than lazy.  As a result, the assertions were never firing.